src/components/editMeasure/populationCriteria/QDMReporting/QDMReporting.tsx

```diff
@@ -1,7 +1,175 @@
+import {
+  FieldLabel,
+  FieldSeparator,
+} from "../../../../styles/editMeasure/populationCriteria/supplementalData";
+
+interface ReportingForm {
+  rateAggregation: string;
+}
 
 const QDMReporting = () => {
-  return <div>Reporting for QDM. Coming Soon!! </div>;
+  const [measure, setMeasure] = useState<Measure>(measureStore.state);
+  const [discardDialogOpen, setDiscardDialogOpen] = useState(false);
+  const measureServiceApi = useMeasureServiceApi();
+  const { updateMeasure } = measureStore;
+  // Toast utilities
+  const [toastOpen, setToastOpen] = useState<boolean>(false);
+  const [toastMessage, setToastMessage] = useState<string>("");
+  const [toastType, setToastType] = useState<string>("danger");
+  const { updateRouteHandlerState } = routeHandlerStore;
+
+  useEffect(() => {
+    const subscription = measureStore.subscribe(setMeasure);
+    return () => {
+      subscription.unsubscribe();
+    };
+  }, []);
+  const canEdit = checkUserCanEdit(
+    measure?.createdBy,
+    measure?.acls,
+    measure?.measureMetaData?.draft
+  );
+
+  const INITIAL_VALUES = {
+    rateAggregation: measure?.rateAggregation || "",
+  };
+
+  const formik = useFormik({
+    initialValues: { ...INITIAL_VALUES },
+    enableReinitialize: true,
+    onSubmit: async (values: ReportingForm) => await handleSubmit(values),
+  });
+  const { resetForm } = formik;
+
+  useEffect(() => {
+    updateRouteHandlerState({
+      canTravel: !formik.dirty,
+      pendingRoute: "",
+    });
+  }, [formik.dirty]);
+
+  const onCancel = () => {
+    setDiscardDialogOpen(true);
+  };
+
+  const onToastClose = () => {
+    setToastType("danger");
+    setToastMessage("");
+    setToastOpen(false);
+  };
+  const handleToast = (type, message, open) => {
+    setToastType(type);
+    setToastMessage(message);
+    setToastOpen(open);
+  };
+
+  const handleSubmit = async (values) => {
+    const newMeasure: Measure = {
+      ...measure,
+      rateAggregation: values.rateAggregation,
+    };
+
+    measureServiceApi
+      .updateMeasure(newMeasure)
+      .then(() => {
+        handleToast("success", "Measure Reporting Updated Successfully", true);
+        // updating measure will propagate update state site wide.
+        updateMeasure(newMeasure);
+      })
+      // update to alert
+      .catch((err) => {
+        handleToast(
+          "danger",
+          "Error updating Measure Reporting: " +
+            err?.response?.data?.message?.toString(),
+          true
+        );
+      });
+  };
+
+  return (
+    <MetaDataWrapper
+      header="Reporting"
+      canEdit={canEdit}
+      dirty={formik.dirty}
+      isValid={formik.isValid}
+      handleSubmit={formik.handleSubmit}
+      onCancel={onCancel}
+    >
+      <form
+        id="measure-reporting-form"
+        onSubmit={formik.handleSubmit}
+        data-testid="reporting-form"
+      >
+        <div id="reporting" data-testid="reporting">
```

Comment:
Do you think, we could reuse the existing reporting tab from QiCore by making it a separate component? Unless the functionality between these models is different.

Reply:
As far as I know, it should function the same way, but we can confirm it from POs

Reply:
There's MAT-5553 to do Improvement Notation too.